src/encoder.rs

```diff
@@ -1,7 +1,7 @@
-use types::{Result, OscType, OscPacket, OscBundle, OscMessage};
 use errors::OscError;
+use types::{OscBundle, OscMessage, OscPacket, OscType, Result};
 
-use byteorder::{ByteOrder, BigEndian};
```

Comment:
If it's not too much hassle please try to omit the formatting changes or put them in a separate commit. This makes reviewing a bit easier.

Reply:
Thank you for pointing this out. Seems like there is some auto-formatting going on that I can't turn off (VS Code  and rustfmt?), so I'll be careful when adding changes to Git.